test/prototype_transforms_kernel_infos.py

```diff
@@ -2100,3 +2118,39 @@ def sample_inputs_convert_dtype_video():
         ),
     ]
 )
+
+
+def sample_inputs_uniform_temporal_subsample_video():
```

Comment:
Defining a `sample_inputs_fn` for a `KernelInfo` is the only non-optional thing that you need to define. Per comment

https://github.com/pytorch/vision/blob/9c112935abe400222cca8f9fbc2d8386e0f25e80/test/prototype_transforms_kernel_infos.py#L42-L44

these inputs will be used for the following tests:

- https://github.com/pytorch/vision/blob/9c112935abe400222cca8f9fbc2d8386e0f25e80/test/test_prototype_transforms_functional.py#L92
- https://github.com/pytorch/vision/blob/9c112935abe400222cca8f9fbc2d8386e0f25e80/test/test_prototype_transforms_functional.py#L122
- https://github.com/pytorch/vision/blob/9c112935abe400222cca8f9fbc2d8386e0f25e80/test/test_prototype_transforms_functional.py#L157
- https://github.com/pytorch/vision/blob/9c112935abe400222cca8f9fbc2d8386e0f25e80/test/test_prototype_transforms_functional.py#L170
- https://github.com/pytorch/vision/blob/9c112935abe400222cca8f9fbc2d8386e0f25e80/test/test_prototype_transforms_functional.py#L181

This function should either `yield` individual `ArgsKwargs` or `return` a sequence of them. We have been using the `yield` variant for now since it is less verbose, but it really doesn't matter since the output will be converted to a `list` by the machinery anyway.